cqc/pythonLib.py

```diff
@@ -233,19 +252,21 @@ def __init__(self, name, socket_address=None, cqcFile=None, appFile=None, appID=
         # Classical connections in the application network
         self._classicalConn = {}
 
-        if socket_address is None:
-            # This file defines the network of CQC servers interfacing to virtual quantum nodes
-            if cqcFile is None:
-                self.cqcFile = get_cqc_file()
-            else:
-                self.cqcFile = cqcFile
-
-            # Read configuration files for the cqc network
-            if os.path.exists(self.cqcFile):
-                self._cqcNet = networkConfig(self.cqcFile)
+        if socket_address is None or use_classical_communication:
+            if backend is None or backend == "simulaqron":
+                if _simulaqron_major < 3:
```

Comment:
I don't know if this is the best way to go about this. But this should probably be in simulaqron itself as requirement. V2 simulaqron should have cqc <3.0.0

Reply:
@LeonWubben Why should it be in SimulaQron? Why should simlaqron v2 have cqc<3.0.0, simulaqron v2 works fine with cqc 3 it's the pythonlibrary of cqc 3 which does not work with simulaqron 2.